compiler/stable_mir/src/unstable/convert/stable/mir.rs

```diff
@@ -864,6 +864,7 @@ impl<'tcx> Stable<'tcx> for mir::interpret::GlobalAlloc<'tcx> {
             mir::interpret::GlobalAlloc::Memory(alloc) => {
                 GlobalAlloc::Memory(alloc.stable(tables, cx))
             }
+            mir::interpret::GlobalAlloc::TypeId { .. } => todo!(),
```

Comment:
Is it okay to have a `todo!` here or does this need to be resolved before landing?